### PR TITLE
#181 Ukrainian localization file added

### LINKED ISSUE #181: [Ready to help] Ukrainian and Russian translation
Hi @hovancik and the team,

thanks a lot for creating a health-keeping software!

A bit of contribution from my side -- I'm ready to assist with UA and RU i18n if it's required. Just give me a file with English keys -- I'll come back with UA/RU.

### PR DESCRIPTION
**Pull request by neooleg**
fixes #181

### CODE REVIEW

File: app/locales/ua.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
+{
+  "Main": {
```
Comment (hovancik) [inline app/locales/ua.json]: this should be `"main": {` (not capital M) 

other keys should be like that as well.

**Comment by hovancik** [comment]
Thanks! Few things I forgot: 
- in settings3.html you should add something like this `<option value="ru">Русский</option>` for the UA
- you should add yourself in README, like here: https://github.com/hovancik/stretchly#contributors
- also update CHANGELOG.md

Let me know if you need help with anything

**Comment by neooleg** [comment]
@hovancik thanks the feedback -- I did it.

P.S. I've moved the following line upper in settings3.html to satisfy alphabetical order as it was intended.

```           
<option value="pt-BR">Português Brasileiro</option>
```

**Comment by hovancik** [comment]
I've edited file once again upon trying it, as it needed to be called uk, not ua. 
Thanks for fixing order of languages ;]

**Comment by neooleg** [comment]
Sure 😊

Do you need more actions from my side to merge the PR?

**Comment by hovancik** [comment]
Nope, many thanks! ;]

**Comment by neooleg** [comment]
You too for a massive assistance! 😉

**Pull request merged**